website/docs/docs/available-adapters.md

```diff
@@ -47,7 +47,7 @@ These adapter plugins are built and maintained by the same people who build and
 | Starburst & Trino ([dbt-trino](https://github.com/starburstdata/dbt-trino))                                   | [Profile Setup](trino-profile)                                                                | `pip install dbt-trino` |
 | Teradata ([dbt-teradata](https://github.com/teradata/dbt-teradata))                                           | [Profile Setup](teradata-profile), [Configuration](teradata-configs)                          | `pip install dbt-teradata` |
 | TiDB ([dbt-tidb](https://github.com/pingcap/dbt-tidb))                                                        | [Profile Setup](tidb-profile)                                                                 | `pip install dbt-tidb` |
-
+| Layer ([dbt-layer](https://github.com/layerai/dbt-layer))                                                        | [Profile Setup](layer-profile)     | `pip install dbt-layer-bigquery -U` | 
```

Comment:
random question, do you forsee a line in this table for each of the DWs you intent to support?

Reply:
I will let @volkangurel to chime in but ideally yes. We will have a single repo for all adapters but multiple pypi releases.